### PR TITLE
Fixed Android demo app to show basic plot

### PR DESCRIPTION
**Pull request by boguscoder**

### CODE REVIEW

File: HelloWorld/AndroidApp1/AndroidApp1.csproj [hunk 1]
```diff
@@ -45,14 +45,16 @@
...
-    <Reference Include="OxyPlot, Version=2015.1.689.0, Culture=neutral, PublicKeyToken=638079a8f0bd61e9, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\OxyPlot.Core.2015.1.689-alpha\lib\portable-net4+sl4+wp71+win8+wpa81\OxyPlot.dll</HintPath>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Xml" />
+    <Reference Include="OxyPlot">
+      <HintPath>..\packages\OxyPlot.Core.2015.1.744-alpha\lib\portable-net4+sl5+netcore45+wpa81+wp8+MonoAndroid1+MonoTouch1+Xamarin.iOS10\OxyPlot.dll</HintPath>
```
Comment (objorke) [inline HelloWorld/AndroidApp1/AndroidApp1.csproj]: this looks like an old package. major version number should be 1

File: HelloWorld/AndroidApp1/AndroidApp1.csproj [hunk 2]
```diff
@@ -75,6 +78,12 @@
   <ItemGroup>
     <Content Include="Properties\AndroidManifest.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <XamarinComponentReference Include="oxyplot">
+      <Version>2015.1.744</Version>
```
Comment (objorke) [inline HelloWorld/AndroidApp1/AndroidApp1.csproj]: old version

File: HelloWorld/AndroidApp1/packages.config [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="OxyPlot.Core" version="2015.1.744-alpha" targetFramework="monoandroid60" />
```
Comment (objorke) [inline HelloWorld/AndroidApp1/packages.config]: old version

**Comment by objorke** [comment]
thanks for fixing the demo app! Can you update to the latest packages?

**Comment by boguscoder** [comment]
absolutely, ill ping with update shortly

**Comment by boguscoder** [comment]
Done

**Comment by objorke** [comment]
It still looks wrong, this should be the latest:
https://www.nuget.org/packages/OxyPlot.Core/1.0.0-unstable2100

**Comment by boguscoder** [comment]
Xamarin studio updated to what you see in last commit, are you sure you want 'unstable' there?

**Comment by objorke** [comment]
Yes, all the other packages are unlisted. Don´t know why they show up in Xamarin Studio. Bug?

**Comment by boguscoder** [comment]
I've committed what was available on nudget.org

File: HelloWorld/AndroidApp1/AndroidApp1.csproj [hunk 4]
```diff
@@ -45,14 +45,14 @@
...
-    <Reference Include="OxyPlot, Version=2015.1.689.0, Culture=neutral, PublicKeyToken=638079a8f0bd61e9, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\OxyPlot.Core.2015.1.689-alpha\lib\portable-net4+sl4+wp71+win8+wpa81\OxyPlot.dll</HintPath>
-    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="System.Xml" />
+    <Reference Include="OxyPlot.Xamarin.Android">
+    </Reference>
```
Comment (objorke) [inline HelloWorld/AndroidApp1/AndroidApp1.csproj]: HintPath has been removed manually? will this work?

Comment (boguscoder) [inline HelloWorld/AndroidApp1/AndroidApp1.csproj]: Builds and runs, do you have any chance to try?

Comment (boguscoder) [inline HelloWorld/AndroidApp1/AndroidApp1.csproj]: Actually no, let me push one last change, I dont like the way Xamarin studio 'remembers' things..

**Comment by objorke** [comment]
thanks!

**Pull request merged**